src/transformers/commands/env.py

```diff
@@ -132,6 +134,11 @@ def run(self):
             "Using GPU in script?": "<fill in>",
             "Using distributed or parallel set-up in script?": "<fill in>",
         }
+        if pt_cuda_available:
+            info["GPU type"] = torch.cuda.get_device_name()
```

Comment:
Could you also move the line `"Using GPU in script?": "<fill in>",` to be conditional inside these if/else options?

Reply:
done :-) @amyeroberts

Reply:
Ah, I should have been more specific. There should be an equivalent "Using NPU in script?" added too

Reply:
Added "Using GPU in script?", please take another look at this PR, thanks.